config/mongoid.yml

```diff
@@ -12,7 +12,6 @@ test:
 
 production:
   database: govuk_content_production
-  refresh_mode: :sync
```

Comment:
This appears to be [another option associated with `Mongo::MongoReplicaSetClient`](https://github.com/mongodb/mongo-ruby-driver/blob/1.12.1/lib/mongo/mongo_replica_set_client.rb#L101-L104) as per my comment on the previous commit.